src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -30,33 +30,37 @@ public HttpConnectionDispatcher(ConnectionManager manager, ILoggerFactory logger
 
         public async Task ExecuteAsync(HttpContext context, HttpSocketOptions options, SocketDelegate socketDelegate)
         {
-            if (!await AuthorizeHelper.AuthorizeAsync(context, options.AuthorizationData))
+            var logScope = new ConnectionLogScope(context.Request.Query["id"]);
```

Comment:
Add a comment explaining this. Maybe write a helper called GetConnectionId and call it from both places in this file.